web/src/tippyjs.js

```diff
@@ -232,7 +232,7 @@ export function initialize() {
     });
 
     delegate("body", {
-        target: "#stream-specific-notify-table .unmute_stream",
+        target: "#stream-specific-notify-table .unmute_stream, #add_streams_tooltip, #filter_streams_tooltip",
         appendTo: () => document.body,
```

Comment:
Apologies, could you please provide more clarity on what is expected or required in this situation? I am finding it difficult to fully understand. 
I think [this](https://github.com/zulip/zulip/pull/24819/commits/8ba6721a8380632ef08532fd26dade813d516cc7) commit does the thing you might be asking for.

Reply:
I mean we can merge
```js
    delegate("body", {
        target: [
            ".recipient_bar_icon",
            "#streams_header .sidebar-title",
            "#userlist-title",
            "#user_filter_icon",
            "#scroll-to-bottom-button-clickable-area",
            ".code_external_link",
            ".spectator_narrow_login_button",
        ],
        appendTo: () => document.body,
    });
```
 and 
```
    delegate("body", {
        target: "#stream-specific-notify-table .unmute_stream",
        appendTo: () => document.body,
    });
```
code blocks since they do the same thing as a prep commit.